specs/pvs/corla.pvs

```diff
@@ -1309,9 +1399,29 @@ BEGIN
 %%% or other means specified by the Secretary of State.
 
   next_ballot_for_audit:
-    [audit_board_dashboard -> [ballot, audit_board_dashboard]]
+    [audit_board_dashboard ->
+     [ballot_manifest_info, audit_board_dashboard]]
+
+% @trace Per email from Jerome Lovato at CDOS on 19 July 2017: The
+% Audit Board will enter the choices for the entire ballot including
+% overvotes and undervotes. The tool must not allow the Audit Board to
+% skip contests.
   report_markings:
-    [audit_board_dashboard, list[ballot_mark] -> audit_board_dashboard]
+    [audit_board_dashboard, ballot_manifest_info,
+     list[ballot_mark] -> audit_board_dashboard]
+
+% @trace Per email from Jerome Lovato at CDOS on 19 July 2017: If the
+% Audit Board cannot find a ballot, the tool should have a “ballot not
+% found” box that the Audit Board will select. This will be marked as
+% an error and the county may have to audit additional ballots.
+  report_ballot_not_found:
+    [audit_board_dashboard, phantom_ballot -> audit_board_dashboard]  
+
+%%% If a ballot does not have a voter-verifiable paper ballot associated
+%%% with it then we report such and recompute.
```

Comment:
Upon further reflection, I'm thinking `non_voter_verifiable_ballot` should be tracked and reported on, but should not move an audit towards a full hand count. 
This is based on a query from @mkiniry and recent calculations and discussion with CORLA2 (where we may not have unanimity on this). We don't trust the `non_voter_verifiable_ballot`, but we've got nothing better. Trigering escalation would be pointless because a full hand count can't correct anything with respect to these ballots, and the effort would be wasted.

We could just say those ballots should not be included in an audit at all ("don't waste your time on them"), and attach a 
caveat to the final report documenting the number of ballots that were assumed to be correct, but that sounds like an out-of-scope  enhancement request to me at the moment.